lib/linguist/languages.yml

```diff
@@ -6058,15 +6080,12 @@ Text:
   - README.nss
   - click.me
   - delete.me
-  - go.mod
-  - go.sum
   - keep.me
   - package.mask
   - package.use.mask
   - package.use.stable.mask
   - read.me
   - readme.1st
-  - test.me
```

Comment:
Removed this one as it only had [228 files](https://github.com/search?q=filename%3Atest.me+extension%3Ame&type=Code) on GitHub, of which only [110 are relevant](https://github.com/search?q=filename%3Atest.me+extension%3Ame+language%3Atext&type=Code)

Reply:
@Nixinova That's really something that belongs in its own pull-request.